### DISCUSSION TITLE
Save the viewing image after rotating

### DISCUSSION THREAD

**Discussion by loenieki** [d2phap/ImageGlass#1373] [Q&A]
I am using ImageGlass Kobe 8.6.7.13 with Windows 10 and I applied the option "Save the viewing image after rotating".

Every time, when I rotate files with extension .jpg for example and I close ImageGlass, the file is updating with the preference in the previous step.

The size every time is a lot bigger than the original file, before to be rotated. If I use Windows rotate function or other software I don't see this behavior.

So, can someone explain why the files produced with ImageGlass after rotating are so much different in size, compared to the original files or that's a bug?

**Comment by d2phap** [comment]
ImageGlass does not support lossless saving, so I set the default quality to maximum 100.
You can change it in Settings > Edit:

![image](https://user-images.githubusercontent.com/3154213/174779759-e22bffba-bc23-40e1-bebb-ce6d85a903c1.png)

**Comment by loenieki** [comment]
The image quality is already with the default value of 100 in the "Edit" menu but ImageGlass is doubling the size after rotating.

Example, if .jpg file is 5MB, after just rotating and saving it with ImageGlass, the new size is 10MB.

@d2phap Based on your answer, I am finding this option not usable.
